Frameworks/include/NSParagraphStyleInternal.h

```diff
@@ -16,13 +16,36 @@
 
 #pragma once
 
-#import <UIKit/NSParagraphStyle.h>
 #import <UIKit/NSMutableParagraphStyle.h>
+#import <CoreText/CTParagraphStyle.h>
 
-@interface NSParagraphStyle () {
-@protected
-    float _hyphenationFactor;
-    NSTextAlignment _alignment;
-    NSLineBreakMode _lineBreakMode;
-}
+@interface NSParagraphStyle ()
+@property (nonatomic) NSTextAlignment alignment;
```

Comment:
>nonatomic [](start = 11, length = 9)

nit: add a comment that you're redeclaring these as readwrite. you might even want to explicitly state it so that the compiler will have the chance to complain.